cmd/build/v2/config.go

```diff
@@ -67,7 +66,7 @@ func getConfig(registry configRegistryInterface, gitRepo configRepositoryInterfa
 		hasGlobalAccess:     hasAccess,
 		isCleanProject:      isClean,
 		fs:                  afero.NewOsFs(),
-		isOkteto:            okteto.Context().IsOkteto,
```

Comment:
why is this always `true`?

Reply:
I've changed to use global var when build command is not exec, check it!